### PR TITLE
fix: ClientNetworkTransform fix with 1.0.1+ changes main

### PR DESCRIPTION
**Pull request by SamuelBellomo**
### Description

Updating CNT to follow changes with NGO 1.0.1+
This ensure that CNT doesn't send twice the RPCs and relies on the new owner driven NetworkVariable instead.
This will ensure that CNT's updates are now tick based (instead of RPCs which are Update based)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] JIRA ticket ID is in the PR title or at least one commit message
 - [ ] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Multiplayer Samples Co-op Changelog
 
+## [1.3.1-pre] - 2022-09-xx
```
Comment (SamuelBellomo) [inline Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:3]: Needs to be updated before merging

File: Packages/com.unity.multiplayer.samples.coop/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "com.unity.learn.iet-framework": "1.2.1",
     "com.unity.multiplayer.tools": "1.0.0-pre.7",
-    "com.unity.netcode.gameobjects": "1.0.0-pre.10",
+    "com.unity.netcode.gameobjects": "1.0.1",
```
Comment (SamuelBellomo) [inline Packages/com.unity.multiplayer.samples.coop/package.json:12]: Needs to be updated before merging

**Review by NoelStephensUnity — APPROVED** [review]
LGTM!

**Review by pdeschain — APPROVED** [review]

File: Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+
+### Fixed
+* Updating ClientNetworkTransform for latest NGO 1.0.1
+
```
Comment (SamuelBellomo) [inline Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md:7]:
```suggestion
## [1.3.1-pre] - 2022-09-13

### Fixed
* Updating ClientNetworkTransform for latest NGO 1.0.1+ (#726) This fixes the new behaviour where both RPCs and netvars were sending the same data. Now CNT is netvar based only, making it send on tick just like NetworkTransform and other network variables would.

```

**Pull request merged**